src/libtools/graph.cc

```diff
@@ -83,18 +83,18 @@ vector<int> Graph::getVerticesMissingNodes(void) {
   return missing;
 }
 
-vector<pair<int,GraphNode>> Graph::getNeighNodes(int vert){
+vector<pair<int, GraphNode>> Graph::getNeighNodes(int vert) {
   auto neigh_vertices = getNeighVertices(vert);
-  vector<pair<int,GraphNode>> neigh_vertices_pr;
-  for(auto neigh_vert : neigh_vertices){
-    auto node_pr = pair<int,GraphNode>(neigh_vert,nodes_[neigh_vert]);
+  vector<pair<int, GraphNode>> neigh_vertices_pr;
+  for (auto neigh_vert : neigh_vertices) {
+    auto node_pr = pair<int, GraphNode>(neigh_vert, nodes_[neigh_vert]);
     neigh_vertices_pr.push_back(node_pr);
   }
   return neigh_vertices_pr;
 }
 
-void Graph::setNode(int vert, GraphNode gn){
-  if(nodes_.count(vert)){
+void Graph::setNode(int vert, GraphNode gn) {
```

Comment:
should this be a copy, or if graph is not a pdo type this should be a const reference

Reply:
what is 'pdo' ?

Reply:
I'm not fixing this in stable, because it might start me down a rabbit hole and it is not what is causing the error.